src/runtime/config/configuration-qemu-nvidia-gpu.toml.in

```diff
@@ -140,7 +139,7 @@ default_bridges = @DEFBRIDGES@
 
 # Default memory size in MiB for SB/VM.
 # If unspecified then it will be set @DEFMEMSZ@ MiB.
-default_memory = @DEFMEMSZ@
+default_memory = 65536
```

Comment:
In this config can you use the templates for:
```
NVDEFAULTCPUS = 16
NVDEFAULTMEMORY = 65536
NVDEFAULTVFIOPORT = root-port
NVDEFAULTPCIEROOTPORT = 8
NVDEFAULTTIMEOUT = 320
```
for consistency.

Reply:
Good catch missed that.

Reply:
It looks like this hasn't been pushed? We still have the numeric constants in this file e.g:
`default_vcpus = 16`, `default_memory = 65536` etc?

Reply:
Not pushed and missed the `pcie_root_port` setting as well. No numeric constants.